engine/apps/public_api/serializers/incidents.py

```diff
@@ -19,14 +17,6 @@ class IncidentSerializer(EagerLoadingMixin, serializers.ModelSerializer):
     resolved_by = UserIdField(read_only=True, source="resolved_by_user")
 
     SELECT_RELATED = ["channel", "channel_filter", "slack_message", "channel__organization"]
-    PREFETCH_RELATED = [
-        "alerts",
-        Prefetch(
-            "telegram_messages",
-            TelegramMessage.objects.filter(chat_id__startswith="-", message_type=TelegramMessage.ALERT_GROUP_MESSAGE),
-            to_attr="prefetched_telegram_messages",
-        ),
-    ]
```

Comment:
We don't use alerts data now, do not prefetch. Stop prefetching telegram messages for permalinks (internal API is not doing that and it seems it is not helping here).